src/main/kotlin/com/anaplan/engineering/vdmgradleplugin/TypeChecker.kt

```diff
@@ -50,25 +50,29 @@ open class VdmTypeCheckTask(private val includeTests: Boolean) : OvertureTask()
         @InputFiles
         get() = project.locateAllSpecifications(dialect, includeTests)
 
-    val generatedLibFile: File
+    // Only needed to ensure caching of the Gradle task
+    val statusFile: File
```

Comment:
Is it necessary to use a file here? Can we not use a property and let Gradle handle serialize/deserialize?

Reply:
Yes, it is necessary: Gradle only provides file and directory output annotations.